kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -443,6 +475,11 @@ def run_cfg_group(tests: list[str]) -> dict[str, bool]:
     failed = [setup_cfg for setup_cfg, passed in results.items() if not passed]
     if failed:
         raise ValueError(f'Running setUp method failed for {len(failed)} contracts: {failed}')
+    contracts = unique({test.split('.')[0] for test in tests})
+    _LOGGER.info(f'contracts: {contracts}')
```

Comment:
I guess this is for debugging? Can it be removed?

Reply:
Removed